api.raml

```diff
@@ -61,11 +61,23 @@ uses:
         200:
           body:
             application/json: lib.JobResponse
-    /{file}:
-      uriParameters:
-        file:
-          type: string
-          description: MUST be one of "meta", "tabular", "statistics".
+    /tabular:
+      post:
+        body:
+	  application/json: lib.BetaDivPluginRequest
+	responses:
+	  200:
+            body:
+	      text/tab-separated-values: lib.EntityTabularPostResponse    
```

Comment:
idk. i get the metadata response and the empty response.. but the request and the stats responses when not empty will all be really specific to the plugin. is that still worth put in edacommon?

Reply:
Well, it would be nice if the data service could deserialize the JSON it gets from the stats endpoint back into a Java object for use by the viz plugin.  It would need to have direct access to that class or to the RAML to do that.  I'm thinking maybe I'll just read it directly from the compute raml though.  Means a dependency from data svc -> compute svc (will have to keep version up to date to avoid issues).  So to prevent problems, maybe better to put stuff in EdaCommon, then update the eda common version in both data and compute.  We typically update the common version in all services when it iterates so would be harder to forget to do that than to update a compute version in data svc.

Reply:
ok. i dont have anything using the stats endpoint YET. (i anticipate heatmaps/ correlograms will be the first). does that mean its ok as it is for now? or do you want the requests in edacommon too?